Failures in [node-test-pull-request/61618](https://ci.nodejs.org/job/node-test-pull-request/61618/) to [node-test-pull-request/61713](https://ci.nodejs.org/job/node-test-pull-request/61713/) that failed 2 or more PRs
(Generated with `ncu-ci walk pr --stats=true --markdown reports/2024-08-30.md`)

| UTC Time         | RUNNING | SUCCESS | UNSTABLE | ABORTED | FAILURE | Green Rate |
| ---------------- | ------- | ------- | -------- | ------- | ------- | ---------- |
| 2024-08-30 17:13 | 9       | 4       | 17       | 3       | 67      | 4.55%      |


### Jenkins Failure

| Reason | <code>Backing channel 'JNLP4-connect connection from ... is disconnected.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 8 ([https://github.com/nodejs/node/pull/54411/](https://ci.nodejs.org/job/node-test-pull-request/61630/), [https://github.com/nodejs/node/pull/54571/](https://ci.nodejs.org/job/node-test-pull-request/61635/), [https://github.com/nodejs/node/pull/54620/](https://ci.nodejs.org/job/node-test-pull-request/61638/), [https://github.com/nodejs/node/pull/54512/](https://ci.nodejs.org/job/node-test-pull-request/61643/), [https://github.com/nodejs/node/pull/54367/](https://ci.nodejs.org/job/node-test-pull-request/61661/), [https://github.com/nodejs/node/pull/54413/](https://ci.nodejs.org/job/node-test-pull-request/61663/), [https://github.com/nodejs/node/pull/54609/](https://ci.nodejs.org/job/node-test-pull-request/61668/), [https://github.com/nodejs/node/pull/54510/](https://ci.nodejs.org/job/node-test-pull-request/61669/)) |
| Appeared | [undefined](https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/37972/console), [test-digitalocean-ubuntu2204-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60048/console), [test-rackspace-win2022_vs2022-x64-4](https://ci.nodejs.org/job/node-compile-windows/nodes=win-vs2022-arm64/57794/console), [test-digitalocean-ubuntu2204-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60019/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61630/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61669/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/37972/console">Example</a></summary>

```
java.io.IOException: Backing channel 'JNLP4-connect connection from 138.197.4.1/138.197.4.1:53022' is disconnected.
	at hudson.remoting.RemoteInvocationHandler.channelOrFail(RemoteInvocationHandler.java:215)
	at hudson.remoting.RemoteInvocationHandler.invoke(RemoteInvocationHandler.java:285)
	at jdk.proxy2/jdk.proxy2.$Proxy97.isAlive(Unknown Source)
	at hudson.Launcher$RemoteLauncher$ProcImpl.isAlive(Launcher.java:1212)
	at hudson.Launcher$RemoteLauncher$ProcImpl.join(Launcher.java:1204)
```
</details>

-------

| Reason | <code>Failed to create a temp file on /Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 6 ([https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61628/), [https://github.com/nodejs/node/pull/54209/](https://ci.nodejs.org/job/node-test-pull-request/61637/), [https://github.com/nodejs/node/pull/54620/](https://ci.nodejs.org/job/node-test-pull-request/61638/), [https://github.com/nodejs/node/pull/53774/](https://ci.nodejs.org/job/node-test-pull-request/61640/), [https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54512/](https://ci.nodejs.org/job/node-test-pull-request/61643/)) |
| Appeared | [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60846/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61628/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61643/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60846/console">Example</a></summary>

```
java.io.IOException: Failed to create a temp file on /Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64
	at hudson.FilePath.createTextTempFile(FilePath.java:1661)
	at hudson.tasks.CommandInterpreter.createScriptFile(CommandInterpreter.java:202)
	at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:120)
	at hudson.tasks.CommandInterpreter.perform(CommandInterpreter.java:92)
	at PluginClassLoader for postbuild-task//hudson.plugins.postbuildtask.PostbuildTask.perform(PostbuildTask.java:123)
```
</details>

-------

| Reason | <code>Build timed out (after 60 minutes). Marking the build as failed.</code> |
| - | :- |
| Type | JENKINS_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54380/](https://ci.nodejs.org/job/node-test-pull-request/61618/), [https://github.com/nodejs/node/pull/54599/](https://ci.nodejs.org/job/node-test-pull-request/61622/)) |
| Appeared | [test-ibm-ubi81_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/45727/console), [test-digitalocean-ubi81_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/45724/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61618/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61622/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/45727/console">Example</a></summary>

```
Build timed out (after 60 minutes). Marking the build as failed.
```
</details>

-------


### Git Failure


### Build Failure

| Reason | <code>fatal error: Killed signal terminated program cc1plus</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 11 ([https://github.com/nodejs/node/pull/54553/](https://ci.nodejs.org/job/node-test-pull-request/61633/), [https://github.com/nodejs/node/pull/54571/](https://ci.nodejs.org/job/node-test-pull-request/61635/), [https://github.com/nodejs/node/pull/54620/](https://ci.nodejs.org/job/node-test-pull-request/61638/), [https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54512/](https://ci.nodejs.org/job/node-test-pull-request/61643/), [https://github.com/nodejs/node/pull/54522/](https://ci.nodejs.org/job/node-test-pull-request/61654/), [https://github.com/nodejs/node/pull/54633/](https://ci.nodejs.org/job/node-test-pull-request/61658/), [https://github.com/nodejs/node/pull/54609/](https://ci.nodejs.org/job/node-test-pull-request/61660/), [https://github.com/nodejs/node/pull/54610/](https://ci.nodejs.org/job/node-test-pull-request/61662/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61666/), [https://github.com/nodejs/node/pull/54510/](https://ci.nodejs.org/job/node-test-pull-request/61669/)) |
| Appeared | [test-digitalocean-ubuntu2204-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60049/console), [test-digitalocean-ubuntu2204-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60046/console), [undefined](https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/37966/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61633/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61669/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60049/console">Example</a></summary>

```
fatal error: Killed signal terminated program cc1plus
```
</details>

-------

| Reason | <code>ERROR: Step ‘Record fingerprints of files to track usage’ failed: no workspace for ...</code> |
| - | :- |
| Type | BUILD_FAILURE |
| Failed PR | 7 ([https://github.com/nodejs/node/pull/54411/](https://ci.nodejs.org/job/node-test-pull-request/61629/), [https://github.com/nodejs/node/pull/54610/](https://ci.nodejs.org/job/node-test-pull-request/61634/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61651/), [https://github.com/nodejs/node/pull/54553/](https://ci.nodejs.org/job/node-test-pull-request/61652/), [https://github.com/nodejs/node/pull/52295/](https://ci.nodejs.org/job/node-test-pull-request/61656/), [https://github.com/nodejs/node/pull/54609/](https://ci.nodejs.org/job/node-test-pull-request/61660/), [https://github.com/nodejs/node/pull/54413/](https://ci.nodejs.org/job/node-test-pull-request/61663/)) |
| Appeared | [test-digitalocean-ubuntu2204-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60043/console), [test-digitalocean-ubuntu2204-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60032/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61629/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61663/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=ubuntu2204-64/60043/console">Example</a></summary>

```
ERROR: Step ‘Record fingerprints of files to track usage’ failed: no workspace for node-test-commit-linux/nodes=ubuntu2204-64 #60043
Collecting metadata...
Metadata collection done.
Notifying upstream projects of job completion
Finished: FAILURE
```
</details>

-------


### undefined


### JSTest Failure

| Reason | <code>parallel/test-http2-socket-close</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 13 ([https://github.com/nodejs/node/pull/54209/](https://ci.nodejs.org/job/node-test-pull-request/61620/), [https://github.com/nodejs/node/pull/53571/](https://ci.nodejs.org/job/node-test-pull-request/61631/), [https://github.com/nodejs/node/pull/54512/](https://ci.nodejs.org/job/node-test-pull-request/61643/), [https://github.com/nodejs/node/pull/54627/](https://ci.nodejs.org/job/node-test-pull-request/61645/), [https://github.com/nodejs/node/pull/54569/](https://ci.nodejs.org/job/node-test-pull-request/61659/), [https://github.com/nodejs/node/pull/54413/](https://ci.nodejs.org/job/node-test-pull-request/61663/), [https://github.com/nodejs/node/pull/54581/](https://ci.nodejs.org/job/node-test-pull-request/61667/), [https://github.com/nodejs/node/pull/54609/](https://ci.nodejs.org/job/node-test-pull-request/61678/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61680/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61690/), [https://github.com/nodejs/node/pull/54653/](https://ci.nodejs.org/job/node-test-pull-request/61704/), [https://github.com/nodejs/node/pull/54571/](https://ci.nodejs.org/job/node-test-pull-request/61705/), [https://github.com/nodejs/node/pull/54510/](https://ci.nodejs.org/job/node-test-pull-request/61713/)) |
| Appeared | [test-azure_msft-win11_vs2022-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2019/29741/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/29734/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61620/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61713/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2019/29741/console">Example</a></summary>

```
not ok 484 parallel/test-http2-socket-close
  ---
  duration_ms: 1786.03000
  severity: fail
  exitcode: 1
  stack: |-
    node:events:497
          throw er; // Unhandled 'error' event
          ^
    
    Error: read ECONNRESET
        at TLSWrap.onStreamRead (node:internal/stream_base_commons:218:20)
    Emitted 'error' event on ClientHttp2Stream instance at:
        at emitErrorNT (node:internal/streams/destroy:169:8)
        at emitErrorCloseNT (node:internal/streams/destroy:128:3)
        at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
      errno: -4077,
      code: 'ECONNRESET',
      syscall: 'read'
    }
    
    Node.js v20.17.1-pre
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-run-watch</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 13 ([https://github.com/nodejs/node/pull/54585/](https://ci.nodejs.org/job/node-test-pull-request/61625/), [https://github.com/nodejs/node/pull/54622/](https://ci.nodejs.org/job/node-test-pull-request/61632/), [https://github.com/nodejs/node/pull/54553/](https://ci.nodejs.org/job/node-test-pull-request/61633/), [https://github.com/nodejs/node/pull/54620/](https://ci.nodejs.org/job/node-test-pull-request/61638/), [https://github.com/nodejs/node/pull/54569/](https://ci.nodejs.org/job/node-test-pull-request/61655/), [https://github.com/nodejs/node/pull/54581/](https://ci.nodejs.org/job/node-test-pull-request/61667/), [https://github.com/nodejs/node/pull/54609/](https://ci.nodejs.org/job/node-test-pull-request/61668/), [https://github.com/nodejs/node/pull/52295/](https://ci.nodejs.org/job/node-test-pull-request/61679/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/), [https://github.com/nodejs/node/pull/54643/](https://ci.nodejs.org/job/node-test-pull-request/61684/), [https://github.com/nodejs/node/pull/54642/](https://ci.nodejs.org/job/node-test-pull-request/61685/), [https://github.com/nodejs/node/pull/54605/](https://ci.nodejs.org/job/node-test-pull-request/61686/), [https://github.com/nodejs/node/pull/54510/](https://ci.nodejs.org/job/node-test-pull-request/61713/)) |
| Appeared | [test-azure_msft-win11_vs2022-x64-3](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-COMPILED_BY-vs2022/29749/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win10-COMPILED_BY-vs2022/29729/console), [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-equinix_mnx-smartos20-x64-3](https://ci.nodejs.org/job/node-test-commit-smartos/nodes=smartos20-64/56387/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-4](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_shared_x64/45733/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61625/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61713/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win11-COMPILED_BY-vs2022/29749/console">Example</a></summary>

```
not ok 925 parallel/test-runner-run-watch
  ---
  duration_ms: 121895.01200
  severity: fail
  exitcode: 1
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-esm-loader-hooks-inspect-wait</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 10 ([https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61628/), [https://github.com/nodejs/node/pull/54553/](https://ci.nodejs.org/job/node-test-pull-request/61633/), [https://github.com/nodejs/node/pull/54620/](https://ci.nodejs.org/job/node-test-pull-request/61638/), [https://github.com/nodejs/node/pull/54592/](https://ci.nodejs.org/job/node-test-pull-request/61665/), [https://github.com/nodejs/node/pull/54581/](https://ci.nodejs.org/job/node-test-pull-request/61667/), [https://github.com/nodejs/node/pull/52295/](https://ci.nodejs.org/job/node-test-pull-request/61673/), [https://github.com/nodejs/node/pull/54609/](https://ci.nodejs.org/job/node-test-pull-request/61678/), [https://github.com/nodejs/node/pull/54643/](https://ci.nodejs.org/job/node-test-pull-request/61684/), [https://github.com/nodejs/node/pull/54642/](https://ci.nodejs.org/job/node-test-pull-request/61685/), [https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61687/)) |
| Appeared | [test-digitalocean-rhel9-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=rhel9-x64/60065/console), [test-digitalocean-fedora40-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60063/console), [test-ibm-ubi81_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/45767/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60051/console), [test-digitalocean-ubuntu2204_sharedlibs_container-x64-3](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_icu_x64/45757/console), [test-ibm-alpine319_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=alpine-latest-x64/60021/console), [test-ibm-rhel9-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=rhel9-x64/60017/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-2](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl111_x64/45734/console), [test-ibm-ubuntu2204_sharedlibs_container-x64-1](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_openssl30_x64/45734/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/29686/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61628/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61687/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=rhel9-x64/60065/console">Example</a></summary>

```
not ok 1041 parallel/test-esm-loader-hooks-inspect-wait
  ---
  duration_ms: 516.96300
  severity: fail
  exitcode: 1
  stack: |-
    [test] Connecting to a child Node process
    [test] Testing /json/list
    [err] Debugger listening on ws://127.0.0.1:44135/928a626c-7a32-43f7-b4a0-3b7c70a40d9b
    [err] 
    [err] For help, see: https://nodejs.org/en/docs/inspector
    [err] 
    [err] Debugger attached.
    [err] 
    [err] Waiting for the debugger to disconnect...
    [err] 
    [err] child process crashed, signal SIGSEGV
    AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
    
    null !== 0
    
        at runTest (/home/iojs/build/workspace/node-test-commit-linux/test/parallel/test-esm-loader-hooks-inspect-wait.js:30:10)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5) {
      generatedMessage: true,
      code: 'ERR_ASSERTION',
      actual: null,
      expected: 0,
      operator: 'strictEqual'
    }
    1
  ...

```
</details>

-------

| Reason | <code>parallel/test-macos-app-sandbox</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 6 ([https://github.com/nodejs/node/pull/54380/](https://ci.nodejs.org/job/node-test-pull-request/61618/), [https://github.com/nodejs/node/pull/54599/](https://ci.nodejs.org/job/node-test-pull-request/61622/), [https://github.com/nodejs/node/pull/54624/](https://ci.nodejs.org/job/node-test-pull-request/61624/), [https://github.com/nodejs/node/pull/54643/](https://ci.nodejs.org/job/node-test-pull-request/61684/), [https://github.com/nodejs/node/pull/54605/](https://ci.nodejs.org/job/node-test-pull-request/61686/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61692/)) |
| Appeared | [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60882/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61618/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61692/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60882/console">Example</a></summary>

```
not ok 1939 parallel/test-macos-app-sandbox
  ---
  duration_ms: 7353.49500
  severity: fail
  exitcode: 1
  stack: |-
    node:assert:126
      throw new AssertionError(obj);
      ^
    
    AssertionError [ERR_ASSERTION]: Expected values to be strictly equal:
    
    1 !== 0
    
        at Object.<anonymous> (/Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/test/parallel/test-macos-app-sandbox.js:47:8)
        at Module._compile (node:internal/modules/cjs/loader:1546:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
        at Module.load (node:internal/modules/cjs/loader:1317:32)
        at Module._load (node:internal/modules/cjs/loader:1127:12)
        at TracingChannel.traceSync (node:diagnostics_channel:315:14)
        at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
        at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:166:5)
        at node:internal/main/run_main_module:30:49 {
      generatedMessage: true,
 ...
```
</details>

-------

| Reason | <code>pummel/test-fs-readfile-tostring-fail</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 5 ([https://github.com/nodejs/node/pull/54380/](https://ci.nodejs.org/job/node-test-pull-request/61671/), [https://github.com/nodejs/node/pull/52295/](https://ci.nodejs.org/job/node-test-pull-request/61673/), [https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61687/), [https://github.com/nodejs/node/pull/54605/](https://ci.nodejs.org/job/node-test-pull-request/61693/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61694/)) |
| Appeared | [test-orka-macos11-x64-1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60885/console), [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60871/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61671/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61694/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60885/console">Example</a></summary>

```
not ok 3860 pummel/test-fs-readfile-tostring-fail
  ---
  duration_ms: 216.71600
  severity: fail
  exitcode: 7
  stack: |-
    /Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/test/pummel/test-fs-readfile-tostring-fail.js:77
      throw err;
      ^
    
    AssertionError [ERR_ASSERTION]: ifError got unwanted exception: ENOSPC: no space left on device, write
        at Object.callback (/Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/test/pummel/test-fs-readfile-tostring-fail.js:35:37)
        at callback (node:internal/streams/writable:764:21)
        at onwriteError (node:internal/streams/writable:603:3)
        at onwrite (node:internal/streams/writable:647:7)
        at node:internal/fs/streams:513:5
        at node:internal/fs/streams:455:14
        at FSReqCallback.wrapper [as oncomplete] (node:fs:942:5) {
      generatedMessage: false,
      code: 'ERR_ASSERTION',
      actual: [Error: ENOSPC: no space left on device, write] {
        errno: -28,
        code: 'ENOSPC'...
```
</details>

-------

| Reason | <code>parallel/test-file-write-stream4</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/54380/](https://ci.nodejs.org/job/node-test-pull-request/61618/), [https://github.com/nodejs/node/pull/54624/](https://ci.nodejs.org/job/node-test-pull-request/61624/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61690/), [https://github.com/nodejs/node/pull/54605/](https://ci.nodejs.org/job/node-test-pull-request/61693/)) |
| Appeared | [test-orka-macos11-x64-1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60883/console), [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60881/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61618/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61693/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60883/console">Example</a></summary>

```
not ok 1396 parallel/test-file-write-stream4
  ---
  duration_ms: 120101.09100
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-module-loading-globalpaths</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/54380/](https://ci.nodejs.org/job/node-test-pull-request/61618/), [https://github.com/nodejs/node/pull/54599/](https://ci.nodejs.org/job/node-test-pull-request/61622/), [https://github.com/nodejs/node/pull/54624/](https://ci.nodejs.org/job/node-test-pull-request/61624/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61690/)) |
| Appeared | [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60881/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61618/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61690/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60881/console">Example</a></summary>

```
not ok 1913 parallel/test-module-loading-globalpaths
  ---
  duration_ms: 2393.91700
  severity: fail
  exitcode: 1
  stack: |-
    node:fs:3004
      binding.copyFile(
              ^
    
    Error: ENOSPC: no space left on device, copyfile '/Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/out/Release/node' -> '/Users/iojs/node-tmp/.tmp.1915/install/bin/node'
        at Object.copyFileSync (node:fs:3004:11)
        at Object.<anonymous> (/Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/test/parallel/test-module-loading-globalpaths.js:32:6)
        at Module._compile (node:internal/modules/cjs/loader:1546:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
        at Module.load (node:internal/modules/cjs/loader:1317:32)
        at Module._load (node:internal/modules/cjs/loader:1127:12)
        at TracingChannel.traceSync (node:diagnostics_channel:315:14)
        at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
        at Function.execut...
```
</details>

-------

| Reason | <code>parallel/test-runner-watch-mode-complex</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/54569/](https://ci.nodejs.org/job/node-test-pull-request/61659/), [https://github.com/nodejs/node/pull/54581/](https://ci.nodejs.org/job/node-test-pull-request/61667/), [https://github.com/nodejs/node/pull/54609/](https://ci.nodejs.org/job/node-test-pull-request/61678/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-azure_msft-win10_vs2019-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win10-COMPILED_BY-vs2022/29726/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61659/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2752 parallel/test-runner-watch-mode-complex
  ---
  duration_ms: 1913.91300
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>sequential/test-http-server-request-timeouts-mixed</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 4 ([https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61646/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61651/), [https://github.com/nodejs/node/pull/54367/](https://ci.nodejs.org/job/node-test-pull-request/61661/), [https://github.com/nodejs/node/pull/52295/](https://ci.nodejs.org/job/node-test-pull-request/61673/)) |
| Appeared | [test-orka-macos11-x64-1](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60872/console), [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60849/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61646/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61673/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60872/console">Example</a></summary>

```
not ok 4177 sequential/test-http-server-request-timeouts-mixed
  ---
  duration_ms: 3342.59400
  severity: fail
  exitcode: 1
  stack: |-
    node:assert:408
        throw err;
        ^
    
    AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
    
      assert(request2.completed)
    
        at Timeout._onTimeout (/Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/test/sequential/test-http-server-request-timeouts-mixed.js:108:5)
        at listOnTimeout (node:internal/timers:614:17)
        at process.processTimers (node:internal/timers:549:7) {
      generatedMessage: true,
      code: 'ERR_ASSERTION',
      actual: false,
      expected: true,
      operator: '=='
    }
    
    Node.js v23.0.0-pre
  ...

```
</details>

-------

| Reason | <code>parallel/test-child-process-fork-exec-path</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 3 ([https://github.com/nodejs/node/pull/54380/](https://ci.nodejs.org/job/node-test-pull-request/61618/), [https://github.com/nodejs/node/pull/54624/](https://ci.nodejs.org/job/node-test-pull-request/61624/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61690/)) |
| Appeared | [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60881/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61618/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61690/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60881/console">Example</a></summary>

```
not ok 200 parallel/test-child-process-fork-exec-path
  ---
  duration_ms: 3588.49600
  severity: fail
  exitcode: 1
  stack: |-
    node:fs:3004
      binding.copyFile(
              ^
    
    Error: ENOSPC: no space left on device, copyfile '/Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/out/Release/node' -> '/Users/iojs/node-tmp/.tmp.185/node-copy.exe'
        at Object.copyFileSync (node:fs:3004:11)
        at Object.<anonymous> (/Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/test/parallel/test-child-process-fork-exec-path.js:50:4)
        at Module._compile (node:internal/modules/cjs/loader:1546:14)
        at Module._extensions..js (node:internal/modules/cjs/loader:1691:10)
        at Module.load (node:internal/modules/cjs/loader:1317:32)
        at Module._load (node:internal/modules/cjs/loader:1127:12)
        at TracingChannel.traceSync (node:diagnostics_channel:315:14)
        at wrapModuleLoad (node:internal/modules/cjs/loader:217:24)
        at Function.execute...
```
</details>

-------

| Reason | <code>parallel/test-http-get-pipeline-problem</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 3 ([https://github.com/nodejs/node/pull/54512/](https://ci.nodejs.org/job/node-test-pull-request/61643/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61651/), [https://github.com/nodejs/node/pull/54510/](https://ci.nodejs.org/job/node-test-pull-request/61697/)) |
| Appeared | [test-azure_msft-win2016_vs2017-x64-1](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2019-x86/29737/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61643/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61697/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win2016-COMPILED_BY-vs2019-x86/29737/console">Example</a></summary>

```
not ok 351 parallel/test-http-get-pipeline-problem
  ---
  duration_ms: 287.99800
  severity: fail
  exitcode: 3221226505
  stack: |-
    image.length = 45658
    recv 0
    done 0
    recv 1
    recv 2
    done 1
    done 2
    recv 3
    done 3
    recv 4
    done 4
    recv 5
    done 5
    recv 6
    done 6
    recv 7
    done 7
    recv 8
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-output</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 3 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61694/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60071/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61694/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60071/console">Example</a></summary>

```
not ok 2734 parallel/test-runner-output
  ---
  duration_ms: 4873.85800
  severity: fail
  exitcode: 1
  stack: |-
    ▶ test runner output
      ✔ test-runner/output/abort.js (2988.292121ms)
      ✔ test-runner/output/abort-runs-after-hook.js (2944.848397ms)
      ✔ test-runner/output/abort_suite.js (2935.963896ms)
      ✔ test-runner/output/abort_hooks.js (2930.558881ms)
      ✔ test-runner/output/describe_it.js (3021.847134ms)
      ✔ test-runner/output/describe_nested.js (2909.563875ms)
      ✔ test-runner/output/eval_dot.js (2961.356723ms)
      ✔ test-runner/output/eval_spec.js (2884.532076ms)
      ✔ test-runner/output/eval_tap.js (2864.670796ms)
      ✔ test-runner/output/filtered-suite-delayed-build.js (2855.540648ms)
      ✔ test-runner/output/filtered-suite-order.mjs (2842.926318ms)
      ✔ test-runner/output/filtered-suite-throws.js (2858.133809ms)
      ✔ test-runner/output/hooks.js (3394.610089ms)
      ✔ test-runner/output/hooks_spec_reporter.js (3685.645375ms)
      ✔ test-runner/output/skip-e...
```
</details>

-------

| Reason | <code>wpt/test-structured-clone</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 3 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54512/](https://ci.nodejs.org/job/node-test-pull-request/61643/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-azure_msft-win11_vs2022-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=0,nodes=win11-COMPILED_BY-vs2019/29699/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4274 wpt/test-structured-clone
  ---
  duration_ms: 170.04900
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-debugger-heap-profiler</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61646/), [https://github.com/nodejs/node/pull/54510/](https://ci.nodejs.org/job/node-test-pull-request/61669/)) |
| Appeared | [test-equinix-ubuntu2204_container-armv7l-2](https://ci.nodejs.org/job/node-test-binary-armv7l/RUN_SUBSET=js,nodes=ubuntu2204-armv7l/13268/console), [test-rackspace-win2022_vs2022-x64-6](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/29696/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61646/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61669/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-armv7l/RUN_SUBSET=js,nodes=ubuntu2204-armv7l/13268/console">Example</a></summary>

```
not ok 3850 parallel/test-debugger-heap-profiler
  ---
  duration_ms: 360065.42200
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-fs-utimes</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54620/](https://ci.nodejs.org/job/node-test-pull-request/61638/), [https://github.com/nodejs/node/pull/54633/](https://ci.nodejs.org/job/node-test-pull-request/61658/)) |
| Appeared | [test-osuosl-rhel8-ppc64_le-2](https://ci.nodejs.org/job/node-test-commit-plinux/nodes=rhel8-ppc64le/55122/console), [test-osuosl-rhel8-ppc64_le-3](https://ci.nodejs.org/job/node-test-commit-plinux/nodes=rhel8-ppc64le/55111/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61638/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61658/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-plinux/nodes=rhel8-ppc64le/55122/console">Example</a></summary>

```
not ok 1362 parallel/test-fs-utimes
  ---
  duration_ms: 128630.70400
  severity: fail
  exitcode: -15
  stack: |-
    timeout
  ...

```
</details>

-------

| Reason | <code>parallel/test-performance-function</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/52295/](https://ci.nodejs.org/job/node-test-pull-request/61626/), [https://github.com/nodejs/node/pull/54569/](https://ci.nodejs.org/job/node-test-pull-request/61659/)) |
| Appeared | [undefined](https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/37965/console), [test-ibm-rhel8-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=rhel8-x64/60012/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61626/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61659/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-custom-suites-freestyle/37965/console">Example</a></summary>

```
not ok 2472 parallel/test-performance-function
  ---
  duration_ms: 211.11400
  severity: fail
  exitcode: 1
  stack: |-
    
    node:internal/event_target:1090
      process.nextTick(() => { throw err; });
                               ^
    RangeError [ERR_OUT_OF_RANGE]: The value of "val" is out of range. It must be >= 1 && <= 9007199254740991. Received 0
        at RecordableHistogram.record (node:internal/histogram:291:5)
        at processComplete (node:internal/perf/timerify:40:15)
        at timerified m (node:internal/perf/timerify:87:5)
        at /home/iojs/build/workspace/node-test-commit-custom-suites-freestyle/test/parallel/test-performance-function.js:103:5
        at runNextTicks (node:internal/process/task_queues:65:5)
        at listOnTimeout (node:internal/timers:575:9)
        at process.processTimers (node:internal/timers:549:7) {
      code: 'ERR_OUT_OF_RANGE'
    }
    
    Node.js v23.0.0-pre
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-coverage</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61694/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60071/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61694/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60071/console">Example</a></summary>

```
not ok 2719 parallel/test-runner-coverage
  ---
  duration_ms: 3117.29200
  severity: fail
  exitcode: 1
  stack: |-
    ▶ test coverage report
      ✔ handles the inspector not being available (2.13538ms)
    ▶ test coverage report (4.516476ms)
    ▶ test tap coverage reporter
      ✖ coverage is reported and dumped to NODE_V8_COVERAGE if present (153.313034ms)
        AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
        
          assert(result.stdout.toString().includes(report))
        
            at TestContext.<anonymous> (/home/iojs/build/workspace/node-test-commit-linux/test/parallel/test-runner-coverage.js:96:5)
            at Test.runInAsyncScope (node:async_hooks:211:14)
            at Test.run (node:internal/test_runner/test:887:25)
            at Test.start (node:internal/test_runner/test:786:17)
            at TestContext.test (node:internal/test_runner/test:303:20)
            at TestContext.<anonymous> (/home/iojs/build/workspace/node-test-commit-linux/test/paral...
```
</details>

-------

| Reason | <code>parallel/test-runner-coverage-thresholds</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/), [https://github.com/nodejs/node/pull/54606/](https://ci.nodejs.org/job/node-test-pull-request/61694/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60071/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61694/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60071/console">Example</a></summary>

```
not ok 2718 parallel/test-runner-coverage-thresholds
  ---
  duration_ms: 1312.31000
  severity: fail
  exitcode: 1
  stack: |-
    ✖ test passing --test-coverage-lines (118.270703ms)
      AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:
      
        assert(stdout.includes(getTapCoverageFixtureReport()))
      
          at TestContext.<anonymous> (/home/iojs/build/workspace/node-test-commit-linux/test/parallel/test-runner-coverage-thresholds.js:67:5)
          at Test.runInAsyncScope (node:async_hooks:211:14)
          at Test.run (node:internal/test_runner/test:887:25)
          at Test.start (node:internal/test_runner/test:786:17)
          at startSubtestAfterBootstrap (node:internal/test_runner/harness:280:17) {
        generatedMessage: true,
        code: 'ERR_ASSERTION',
        actual: false,
        expected: true,
        operator: '=='
      }
    
    ✖ test failing --test-coverage-lines (111.307834ms)
      AssertionError [ERR_ASSERTION]: The expression evaluated to ...
```
</details>

-------

| Reason | <code>parallel/test-runner-run</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2737 parallel/test-runner-run
  ---
  duration_ms: 2128.52900
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-test-filepath</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2743 parallel/test-runner-test-filepath
  ---
  duration_ms: 88.42000
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-test-fullname</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2744 parallel/test-runner-test-fullname
  ---
  duration_ms: 88.00200
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-typechecking</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2746 parallel/test-runner-typechecking
  ---
  duration_ms: 88.80300
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-runner-v8-deserializer</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2747 parallel/test-runner-v8-deserializer
  ---
  duration_ms: 170.23100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-set-http-max-http-headers</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2754 parallel/test-set-http-max-http-headers
  ---
  duration_ms: 410.37600
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-sqlite</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2806 parallel/test-sqlite
  ---
  duration_ms: 138.60800
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-sqlite-data-types</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2807 parallel/test-sqlite-data-types
  ---
  duration_ms: 266.94600
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-sqlite-database-sync</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2808 parallel/test-sqlite-database-sync
  ---
  duration_ms: 171.89400
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-sqlite-named-parameters</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2809 parallel/test-sqlite-named-parameters
  ---
  duration_ms: 171.28900
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-sqlite-statement-sync</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2810 parallel/test-sqlite-statement-sync
  ---
  duration_ms: 412.48600
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-sqlite-transactions</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2812 parallel/test-sqlite-transactions
  ---
  duration_ms: 137.54600
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-stdout-stderr-write</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 2837 parallel/test-stdout-stderr-write
  ---
  duration_ms: 90.04000
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-stream2-httpclient-response-end</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3018 parallel/test-stream2-httpclient-response-end
  ---
  duration_ms: 112.77000
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-stream2-large-read-stall</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3019 parallel/test-stream2-large-read-stall
  ---
  duration_ms: 90.83700
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-stream2-push</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3023 parallel/test-stream2-push
  ---
  duration_ms: 90.10000
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-stream3-pause-then-read</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3038 parallel/test-stream3-pause-then-read
  ---
  duration_ms: 89.76800
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tcp-wrap-listen</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3052 parallel/test-tcp-wrap-listen
  ---
  duration_ms: 91.16300
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-timers-unref-remove-other-unref-timers</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3110 parallel/test-timers-unref-remove-other-unref-timers
  ---
  duration_ms: 89.96100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-timers-unref-remove-other-unref-timers-only-one-fires</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3111 parallel/test-timers-unref-remove-other-unref-timers-only-one-fires
  ---
  duration_ms: 71.50700
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-cli-max-version-1.2</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3136 parallel/test-tls-cli-max-version-1.2
  ---
  duration_ms: 415.57000
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-cli-max-version-1.3</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3138 parallel/test-tls-cli-max-version-1.3
  ---
  duration_ms: 328.87400
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-cli-min-version-1.0</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3139 parallel/test-tls-cli-min-version-1.0
  ---
  duration_ms: 409.64700
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-cli-min-version-1.1</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3140 parallel/test-tls-cli-min-version-1.1
  ---
  duration_ms: 410.32500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-cli-min-version-1.2</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3142 parallel/test-tls-cli-min-version-1.2
  ---
  duration_ms: 411.97200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-cli-min-version-1.3</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3141 parallel/test-tls-cli-min-version-1.3
  ---
  duration_ms: 408.47100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-client-reject-12</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3151 parallel/test-tls-client-reject-12
  ---
  duration_ms: 172.52900
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-client-resume-12</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3155 parallel/test-tls-client-resume-12
  ---
  duration_ms: 170.46500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-enable-trace</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3188 parallel/test-tls-enable-trace
  ---
  duration_ms: 111.02200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-min-max-version</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3224 parallel/test-tls-min-max-version
  ---
  duration_ms: 409.53200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-pause</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3240 parallel/test-tls-pause
  ---
  duration_ms: 136.77700
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-tls-set-encoding</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3270 parallel/test-tls-set-encoding
  ---
  duration_ms: 170.99200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-trace-events-dynamic-enable-workers-disabled</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3320 parallel/test-trace-events-dynamic-enable-workers-disabled
  ---
  duration_ms: 171.59200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-trace-events-get-category-enabled-buffer</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3325 parallel/test-trace-events-get-category-enabled-buffer
  ---
  duration_ms: 110.24700
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-worker-exit-heapsnapshot</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3628 parallel/test-worker-exit-heapsnapshot
  ---
  duration_ms: 408.97100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-worker-message-not-serializable</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3642 parallel/test-worker-message-not-serializable
  ---
  duration_ms: 137.09000
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-worker-message-port-drain</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3648 parallel/test-worker-message-port-drain
  ---
  duration_ms: 171.13200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>parallel/test-worker-nearheaplimit-deadlock</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54610/](https://ci.nodejs.org/job/node-test-pull-request/61653/), [https://github.com/nodejs/node/pull/54642/](https://ci.nodejs.org/job/node-test-pull-request/61685/)) |
| Appeared | [test-digitalocean-ubuntu2204_sharedlibs_container-x64-9](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_zlib_x64/45768/console), [test-ibm-rhel8-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=rhel8-x64/60034/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61653/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61685/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubuntu2204_sharedlibs_zlib_x64/45768/console">Example</a></summary>

```
not ok 3410 parallel/test-worker-nearheaplimit-deadlock
  ---
  duration_ms: 865.75600
  severity: crashed
  exitcode: -6
  stack: |-
    
    <--- Last few GCs --->
    
    
    <--- JS stacktrace --->
    
    FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory
    ----- Native stack trace -----
    
     1: 0x55ce574813ba node::OOMErrorHandler(char const*, v8::OOMDetails const&) [out/Release/node]
     2: 0x55ce578d7b24 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [out/Release/node]
     3: 0x55ce578d7ef4 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, v8::OOMDetails const&) [out/Release/node]
     4: 0x55ce57b3f48a v8::internal::HeapAllocator::AllocateRawWithRetryOrFailSlowPath(int, v8::internal::AllocationType, v8::internal::AllocationOrigin, v8::internal::AllocationAlignment) [out/Release/node]
     5: 0x55ce57b75dd9 v8::internal::Tagged<v8::internal::HeapObject> v8::internal::LocalHeap::AllocateRaw...
```
</details>

-------

| Reason | <code>parallel/test-worker-track-unmanaged-fds</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3716 parallel/test-worker-track-unmanaged-fds
  ---
  duration_ms: 265.26100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>pseudo-tty/test-stdin-write</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3807 pseudo-tty/test-stdin-write
  ---
  duration_ms: 137.70000
  severity: fail
  exitcode: 120
  stack: |-
  ...

```
</details>

-------

| Reason | <code>pseudo-tty/test-trace-sigint-on-idle</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3811 pseudo-tty/test-trace-sigint-on-idle
  ---
  duration_ms: 328.96400
  severity: fail
  exitcode: 120
  stack: |-
  ...

```
</details>

-------

| Reason | <code>pummel/test-net-pingpong</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3886 pummel/test-net-pingpong
  ---
  duration_ms: 111.47800
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>pummel/test-net-write-callbacks</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3890 pummel/test-net-write-callbacks
  ---
  duration_ms: 1217.57800
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>pummel/test-regress-GH-892</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3894 pummel/test-regress-GH-892
  ---
  duration_ms: 171.14500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>pummel/test-stream-pipe-multi</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3895 pummel/test-stream-pipe-multi
  ---
  duration_ms: 111.37600
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>pummel/test-timers</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-osuosl-ubuntu2204_container-arm64-2](https://ci.nodejs.org/job/node-test-commit-arm/nodes=ubuntu2204-arm64/54412/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3898 pummel/test-timers
  ---
  duration_ms: 1209.62500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>report/test-report-exclude-network</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 3824 report/test-report-exclude-network
  ---
  duration_ms: 171.21200
  severity: fail
  exitcode: 7
  stack: |-
  ...

```
</details>

-------

| Reason | <code>sequential/test-deprecation-flags</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4171 sequential/test-deprecation-flags
  ---
  duration_ms: 268.62500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>sequential/test-http-econnrefused</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4188 sequential/test-http-econnrefused
  ---
  duration_ms: 110.45700
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>sequential/test-net-GH-5504</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4208 sequential/test-net-GH-5504
  ---
  duration_ms: 213.48100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>sequential/test-net-connect-econnrefused</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4210 sequential/test-net-connect-econnrefused
  ---
  duration_ms: 109.76300
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>sequential/test-repl-timeout-throw</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4226 sequential/test-repl-timeout-throw
  ---
  duration_ms: 138.06200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>sequential/test-single-executable-application-empty</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54599/](https://ci.nodejs.org/job/node-test-pull-request/61622/), [https://github.com/nodejs/node/pull/54605/](https://ci.nodejs.org/job/node-test-pull-request/61686/)) |
| Appeared | [test-orka-macos11-x64-2](https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60878/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61622/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61686/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-osx/nodes=osx11-x64/60878/console">Example</a></summary>

```
not ok 4214 sequential/test-single-executable-application-empty
  ---
  duration_ms: 20670.90400
  severity: fail
  exitcode: 1
  stack: |-
    Copied /Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/out/Release/node to /Users/iojs/node-tmp/.tmp.4214/sea
    Injected /Users/iojs/node-tmp/.tmp.4214/sea-prep.blob into /Users/iojs/node-tmp/.tmp.4214/sea
    [process 23234]: --- stderr ---
    error: /Library/Developer/CommandLineTools/usr/bin/codesign_allocate: can't write output file: /Users/iojs/node-tmp/.tmp.4214/sea.cstemp (No space left on device)
    /Users/iojs/node-tmp/.tmp.4214/sea: the codesign_allocate helper tool cannot be found or used
    
    [process 23234]: --- stdout ---
    
    [process 23234]: status = 1, signal = null
    /Users/iojs/build/workspace/node-test-commit-osx/nodes/osx11-x64/test/sequential/test-single-executable-application-empty.js:53
      throw e;
      ^
    
    Error: Cannot sign /Users/iojs/node-tmp/.tmp.4214/sea: Error: - process terminated with status 1,...
```
</details>

-------

| Reason | <code>wasi/test-wasi</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54512/](https://ci.nodejs.org/job/node-test-pull-request/61643/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61651/)) |
| Appeared | [test-azure_msft-win11_vs2022-x64-4](https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-COMPILED_BY-vs2019/29699/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61643/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61651/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win11-COMPILED_BY-vs2019/29699/console">Example</a></summary>

```
not ok 895 wasi/test-wasi
  ---
  duration_ms: 1272.03300
  severity: fail
  exitcode: 1
  stack: |-
    executing cant_dotdot
    (node:3624) ExperimentalWarning: WASI is an experimental feature and might change at any time
    (Use `node --trace-warnings ...` to show where the warning was created)
    fd_prestat_get(3, 69720)
    fd_prestat_dir_name(3, 131088, 8)
    fd_prestat_get(4, 69720)
    fd_prestat_dir_name(4, 131152, 4)
    fd_prestat_get(5, 69720)
    environ_sizes_get(69724, 69720)
    environ_get(138560, 131168)
    fd_fdstat_get(3, 69672)
    path_open(3, 1, 1033, 14, 0, 264240830, 268435455, 0, 69668)
    
    executing clock_getres
    (node:9264) ExperimentalWarning: WASI is an experimental feature and might change at any time
    (Use `node --trace-warnings ...` to show where the warning was created)
    fd_prestat_get(3, 69688)
    fd_prestat_dir_name(3, 131088, 8)
    fd_prestat_get(4, 69688)
    fd_prestat_dir_name(4, 131152, 4)
    fd_prestat_get(5, 69688)
    environ_sizes_get(69692, 6...
```
</details>

-------

| Reason | <code>wpt/test-atob</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4260 wpt/test-atob
  ---
  duration_ms: 137.27500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-blob</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4261 wpt/test-blob
  ---
  duration_ms: 110.15600
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-broadcastchannel</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4262 wpt/test-broadcastchannel
  ---
  duration_ms: 169.33200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-compression</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4263 wpt/test-compression
  ---
  duration_ms: 109.71800
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-console</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4264 wpt/test-console
  ---
  duration_ms: 71.66100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-domexception</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4265 wpt/test-domexception
  ---
  duration_ms: 210.86000
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-encoding</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4266 wpt/test-encoding
  ---
  duration_ms: 137.84500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-events</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4267 wpt/test-events
  ---
  duration_ms: 88.04100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-performance-timeline</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4271 wpt/test-performance-timeline
  ---
  duration_ms: 111.34200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-resource-timing</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4272 wpt/test-resource-timing
  ---
  duration_ms: 110.09800
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-streams</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4273 wpt/test-streams
  ---
  duration_ms: 173.47500
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-timers</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4275 wpt/test-timers
  ---
  duration_ms: 72.35100
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-url</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4276 wpt/test-url
  ---
  duration_ms: 109.40600
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-wasm-webapi</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4278 wpt/test-wasm-webapi
  ---
  duration_ms: 136.23000
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-webcrypto</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4279 wpt/test-webcrypto
  ---
  duration_ms: 170.59200
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

| Reason | <code>wpt/test-webstorage</code> |
| - | :- |
| Type | JS_TEST_FAILURE |
| Failed PR | 2 ([https://github.com/nodejs/node/pull/54575/](https://ci.nodejs.org/job/node-test-pull-request/61641/), [https://github.com/nodejs/node/pull/54591/](https://ci.nodejs.org/job/node-test-pull-request/61683/)) |
| Appeared | [test-digitalocean-fedora40-x64-1](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console), [test-digitalocean-fedora39-x64-2](https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-last-latest-x64/60022/console) |
| First CI | https://ci.nodejs.org/job/node-test-pull-request/61641/ |
| Last CI | https://ci.nodejs.org/job/node-test-pull-request/61683/ |

<details>
<summary><a href="https://ci.nodejs.org/job/node-test-commit-linux/nodes=fedora-latest-x64/60061/console">Example</a></summary>

```
not ok 4280 wpt/test-webstorage
  ---
  duration_ms: 109.34900
  severity: fail
  exitcode: 1
  stack: |-
  ...

```
</details>

-------

